### ISSUE TITLE
Error when using mold on aarch64 hardware (rk3399)

### ISSUE THREAD

**Issue by alex3kov** [rui314/mold#297]
Trying to build [calcurse](https://git.calcurse.org/calcurse.git) with mold 1.0.1 (from ArchARM repos):
```sh
$ mold -run ./configure
configure: error: C compiler cannot create executables
mold: unknown command line option: --fix-cortex-a53-835769
```

**Comment by Alcaro** [comment]
Wasn't that fixed in #285?

1.0.2 was released with that fix earlier today, I guess ArchARM didn't update yet.

**Comment by alex3kov** [comment]
1.0.2 works, thanks @Alcaro
